L06-annotations/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,7 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+        runner.TestRunner.runTests("testcases.SampleTestClass");
```

Comment:
```suggestion
        TestRunner.runTests("testcases.SampleTestClass");
```
Пакет лучше в импорт перенести.